.gitattributes

```diff
@@ -0,0 +1,18 @@
+# Auto detect text files and perform LF normalization
+* text=auto
+
```

Comment:
Should we put this file in a separate commit? Seems useful standalone. (In case for some reason we would need to rollback).